packages/core/test/integration/dialects/postgres/sync-alter-enum.test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const chai = require('chai');
+
+const assert = chai.assert;
+const Support = require('../../support');
+
+const { DataTypes } = require('@sequelize/core');
+
+const dialect = Support.getTestDialect();
+
+if (dialect === 'postgres') {
```

Comment:
Test should run on all dialects, we're trying to delete the `/dialects` test folder.

Can you add this test to https://github.com/sequelize/sequelize/blob/main/packages/core/test/integration/model/sync.test.js instead (and let it run on all dialects)

Reply:
Done.